tests/csrf_tests/tests.py

```diff
@@ -237,7 +237,7 @@ def test_token_node_no_csrf_cookie(self):
         """
         req = self._get_GET_no_csrf_cookie_request()
         resp = token_view(req)
-        self.assertEqual(resp.content, b'')
+        self.assertNotEqual("", resp.content)
```

Comment:
I used the same assert that is used in *test_token_node_empty_csrf_cookie*

I will fix both tests.